content/blog/2022/11/airgap.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Air gap
+authors:
+- Raul Cabello Martin
+date: 2022-11-28
+---
+
+We are glad to announce that Kubewarden can now be installed in air-gapped environments! For that, you will need a 
+private OCI registry accessible by your Kubernetes cluster. Kubewarden policies are WebAssembly modules. Therefore, 
+they can be stored inside an OCI-compliant registry as OCI artifacts. 
+For an air gap installation you need to download all the Kubewarden container images and policies in your workstation, 
+then move them to your private OCI registry. Check the Kubewarden [docs](https://docs.kubewarden.io/operator-manual/airgap/install) 
+for a step-by-step guide.
+
+`kwctl` [v1.3.1](https://github.com/kubewarden/kwctl/releases/tag/v1.3.1) adds two new commands in order to support air gap: 
```

Comment:
```suggestion
We have added new commands to `kwctl` [v1.3.1](https://github.com/kubewarden/kwctl/releases/tag/v1.3.1) in order to simplify air gap deployment: 
```

(trying to reduce passive voice, which the Wordpress linter used to bark at me about quite often)